website/cue/reference/remap/functions/uuid_v7.cue

```diff
@@ -0,0 +1,22 @@
+package metadata
+
+remap: functions: uuid_v7: {
+	category:    "Random"
+	description: """
+		Generates a random [UUIDv7](\(urls.uuidv7)) string.
```

Comment:
Is this the `uuidv7` url defined?

Reply:
Most urls are defined in `website/cue/reference/urls.cue`.

Forgot to mention, you can render these locally with `cd website && make run-production-site-locally`.

Reply:
@pront thx, I've added the missing URL now, see:
https://github.com/vectordotdev/vector/pull/20048/files#diff-d93ed3f0b2ace4e29cca5ab64a8493f2691d3f3351ae7624794a84fd9ab8bdc3R530

Sadly the `cd website && make run-production-site-locally` fails for me locally with the following error message which looks like the upstream is currently broken in that regard:

```
Error: error building site: process: readAndProcessContent: "/vector/website/content/en/docs/reference/configuration/sinks/aws_cloudwatch_metrics.md:1:1": unknown kind "sink" in front matter
```

Reply:
Interesting, I see that error too. I'll look into that.

Reply:
Fixed here: https://github.com/vectordotdev/vector/pull/20058

Reply:
@jszwedko thx, already rebased and works now ... looks as expected in the newly rendered function documentation:

![image](https://github.com/vectordotdev/vector/assets/6830431/fa69b4b3-cff4-4ff3-911c-1b1e9d126358)